src/MooLite/core/inventory/items/DrinkDetail.ts

```diff
@@ -0,0 +1,13 @@
+import { ActionTypeHrid } from "src/MooLite/core/actions/ActionTypeHrid";
+import { ConsumableSlotTypeHrid } from "src/MooLite/core/inventory/ConsumableSlotTypeHrid";
+import { ItemHrid } from "src/MooLite/core/inventory/ItemHrid";
+
+export interface DrinkDetail {
+    actionTypeHrid: ActionTypeHrid;
+    characterID: number;
```

Comment:
I'm purposefully not exposing `characterId` on stuff like chat messages. Here it is your own so there danger, but still